view/base.js

```diff
@@ -378,9 +381,9 @@ if ( typeof kintInitialized === 'undefined' ) {
 			kint.toggle(kintNode);
 			kint.fetchVisiblePluses();
 			return false;
-		} else if ( keyCode === 39 || keyCode === 37 ) { // ARROW LEFT/RIGHT : respectively hides/shows and traverses
+		} else if ( keyCode === 39 || keyCode === 37 || keyCode === 76 || keyCode === 72 ) { // ARROW LEFT/RIGHT : respectively hides/shows and traverses
```

Comment:
I did split it originally, and that's when I realized that you use tabs for indentation.  I didn't really know how to indent such a half line correctly.
I'll give it a try, and you please correct me if I get it wrong.